cmd/podman/ps.go

```diff
@@ -349,22 +345,15 @@ func psCmd(c *cli.Context) error {
 		}
 	} else {
 		// Output Namespace headers
-		fmt.Fprintf(w, "\n%s\t%s\t%s\t%s\t%s\t%s\t%s\t%s\t%s\t%s", hid, hnames, nspid, nscgroup, nsipc, nsmnt, nsnet, nspidns, nsuserns, nsuts)
-	}
-	if len(pss) == 0 {
-		fmt.Fprint(w, "\n")
+		fmt.Fprintf(w, "%s\t%s\t%s\t%s\t%s\t%s\t%s\t%s\t%s\t%s", hid, hnames, nspid, nscgroup, nsipc, nsmnt, nsnet, nspidns, nsuserns, nsuts)
```

Comment:
What changed here? Git says it's different, but I can't see anything...

Reply:
gah, commenting a line works weirdly

Reply:
@mheon leading '\n' was removed.

Reply:
Fascinating: split shows a different (and lesser) diff than unidiff. `git pr 1724 upstream` + `git log -1 -p` shows what I assume is the correct diff, in which `if len(pss) == 0` was chopped.

Reply:
yup leading CR

Reply:
Ack, looks fine then